pom.xml

```diff
@@ -239,15 +239,21 @@
         <dependency>
             <groupId>org.flywaydb</groupId>
             <artifactId>flyway-core</artifactId>
-            <version>9.22.2</version>
+            <version>10.11.0</version>
         </dependency>
 
         <dependency>
             <groupId>org.flywaydb</groupId>
             <artifactId>flyway-mysql</artifactId>
-            <version>9.22.2</version>
+            <version>10.11.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-database-postgresql</artifactId>
+            <version>10.11.0</version>
+            <scope>runtime</scope>
```

Comment:
is this scope needed/on purpose?

Reply:
It is as in instructed in maven, probably not mandatory https://mvnrepository.com/artifact/org.flywaydb/flyway-database-postgresql/10.11.0